src/node/cli.ts

```diff
@@ -556,6 +560,7 @@ export async function setDefaults(cliArgs: UserProvidedArgs, configArgs?: Config
   delete process.env.PASSWORD
   delete process.env.HASHED_PASSWORD
   delete process.env.GITHUB_TOKEN
+  delete process.env.CS_DISABLE_FILE_DOWNLOADS
```

Comment:
I think in this case we actually let it stay.  We delete the others because they are secrets.

Reply:
Ah got it. I wasn't sure if we deleted all but since this isn't sensitive no need to delete.

Reply:
Plus if they run code-server in code-server that will mean the sub-code-server will also get downloads disabled which is probably the desired behavior (of course they could just unset the env var to get around that).